### PR TITLE
GOVUKAPP-572: Voiceover bug for page title

### PR DESCRIPTION
**Pull request by TimGJD**
[Jira Link](https://govukverify.atlassian.net/browse/GOVUKAPP-572)

To resolve bug:

- Post a UIAccessibility 'screen changed' notification every time the continue button is pressed to cause voiceover to focus on the new slide text.
- Combine the title and body slide text as 1 accessibility element as without doing this voiceover gets confused when navigating the slides via horizontal swipe. (I checked with Matt Clear who advised this would be acceptable due to the small amount of text on each slide).

### CODE REVIEW

File: Onboarding/Tests/Mocks/MockAccessibilityPoster.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import Foundation
+import UIKit
+
+@testable import Onboarding
+
+struct MockAccessibilityPoster: AccessibilityPoster {
+    static var _postParams: (UIAccessibility.Notification, Any?)? = nil
```
Comment (batesyuk) [inline Onboarding/Tests/Mocks/MockAccessibilityPoster.swift]: `_receivedPostNotification` & `_receivedPostArgument` please. Tuples can get confusing really quickly.

Comment (TimGJD) [reply Onboarding/Tests/Mocks/MockAccessibilityPoster.swift]: Any objection to a new struct to replace the tuple? Makes more sense in the tests where we need to verify if the function was called or not as we can test if the struct is nil rather than individual vars?

Comment (batesyuk) [reply Onboarding/Tests/Mocks/MockAccessibilityPoster.swift]: I would rather individual vars if testing the vars themselves... it matches elsewhere.
If you are simply testing if something is called have a `var _postCalled: Bool = false` that you set to true if its called.

Comment (TimGJD) [reply Onboarding/Tests/Mocks/MockAccessibilityPoster.swift]: Done

**Review by batesyuk — APPROVED** [review]
